Hazel/src/Hazel/Renderer/GraphicsContext.cpp

```diff
@@ -0,0 +1,21 @@
+#include "hzpch.h"
+#include "Hazel/Renderer/GraphicsContext.h"
+
+#include "Hazel/Renderer/Renderer.h"
+#include "Platform/OpenGL/OpenGLContext.h"
+
+namespace Hazel {
+
+	Scope<GraphicsContext> GraphicsContext::Create(void* window)
+	{
+		switch (Renderer::GetAPI())
+		{
+			case RendererAPI::API::None:    HZ_CORE_ASSERT(false, "RendererAPI::None is currently not supported!"); return nullptr;
+			case RendererAPI::API::OpenGL:  return CreateScope<OpenGLContext>(static_cast<GLFWwindow*>(window));
+		}
+
+		HZ_CORE_ASSERT(false, "Unknown RendererAPI!");
+		return nullptr;
+	}
```

Comment:
Should @TheCherno cover/mention this in a video? Since it's introducing a new file to the project.

Reply:
I agree, is this change necessary? I feel like Cherno should do it himself but it's so small that it could be fine.

Reply:
The only reason this is here is to not have to include OpenGL in the Window class. It is yet another create function that Cherno already rushes over in his videos as it is a million times the same code over and over. Switch over the API's and create the correct one..

As cherno skips over it in his videos, does it really need to be in the videos?